packages/logs/lib/es/helpers.ts

```diff
@@ -62,9 +75,11 @@ export async function migrateMapping() {
                 }
             }
         }
+        return Ok(undefined);
     } catch (err) {
-        logger.error(err);
-        throw new Error('failed_to_init_elasticsearch');
+        const errMsg = err instanceof errors.ConnectionError ? 'failed_to_connect_elasticsearch' : 'failed_to_init_elasticsearch';
```

Comment:
should we also account for TimeoutError? https://github.com/elastic/elasticsearch-js/blob/80044746f0f8cf7ee1c06eb1ff882ae3c5bc76b5/docs/reference/connecting.md?plain=1#L468